pkg/analyzer/analyzers/databricks/requests.go

```diff
@@ -0,0 +1,385 @@
+		/*
+			TODO:
+				- https://docs.databricks.com/api/gcp/workspace/workspace/list (list content inside path)
+				- http://docs.databricks.com/api/gcp/workspace/libraries/allclusterlibrarystatuses (list cluster statuses)
+		*/
+	}
+)
+
+// makeDataBricksRequest send the API request to passed url with passed key as access token and return response body and status code
+func makeDataBricksRequest(client *http.Client, endpoint, token string) ([]byte, int, error) {
+	// create request
+	req, err := http.NewRequest(http.MethodGet, "https://"+endpoint, http.NoBody)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	// add key in the header
+	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", token))
+
+	resp, err := client.Do(req)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	defer func() {
+		_, _ = io.Copy(io.Discard, resp.Body)
+		_ = resp.Body.Close()
+	}()
+
+	responseBodyByte, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, 0, err
+	}
+
+	return responseBodyByte, resp.StatusCode, nil
+}
+
+func captureDataBricksResources(client *http.Client, domain, token string, secretInfo *SecretInfo) error {
+	if err := captureRepos(client, domain, token, secretInfo); err != nil {
+		return err
+	}
+
+	if err := captureGitCreds(client, domain, token, secretInfo); err != nil {
+		return err
+	}
+
+	if err := captureJobs(client, domain, token, secretInfo); err != nil {
+		return err
+	}
+
+	if err := captureClusters(client, domain, token, secretInfo); err != nil {
+		return err
+	}
+
+	if err := captureGroups(client, domain, token, secretInfo); err != nil {
+		return err
+	}
+
+	if err := captureUsers(client, domain, token, secretInfo); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func captureUserInfo(client *http.Client, domain, token string, secretInfo *SecretInfo) error {
+	respBody, statusCode, err := makeDataBricksRequest(client, domain+apiEndpoints[CurrentUser], token)
+	if err != nil {
+		return err
+	}
+
+	switch statusCode {
+	case http.StatusOK:
+		var user CurrentUserInfo
+
+		if err := json.Unmarshal(respBody, &user); err != nil {
+			return err
+		}
+
+		secretInfo.UserInfo = User{
+			ID:       user.ID,
+			UserName: user.UserName,
+		}
+
+		for _, email := range user.Emails {
+			if email.Primary {
+				secretInfo.UserInfo.PrimaryEmail = email.Value
+			}
+		}
+
+		return nil
+	case http.StatusUnauthorized:
+		return fmt.Errorf("invalid/expired personal access token")
+	default:
+		return fmt.Errorf("unexpected status code: %d for API: %s", statusCode, apiEndpoints[CurrentUser])
+	}
+}
+
+func captureTokensInfo(client *http.Client, domain, token string, secretInfo *SecretInfo) error {
+	respBody, statusCode, err := makeDataBricksRequest(client, domain+apiEndpoints[TokensInfo], token)
+	if err != nil {
+		return err
+	}
+
+	switch statusCode {
+	case http.StatusOK:
+		var tokens Tokens
+
+		if err := json.Unmarshal(respBody, &tokens); err != nil {
+			return err
+		}
+
+		for _, token := range tokens.TokensInfo {
```

Comment:
suggestion: Let's consider chaning `token` -> `tok`, or something similar to avoid shadowing the function argument `token`.